main.py

```diff
@@ -0,0 +1,47 @@
+from SLAM.core import Point, Landmark
+from SLAM.landmark_extractor import extract_spike_landmarks
+from SLAM.landmark_matcher import match_landmarks
+from SLAM.position_evaluator import evaluate_position
+from SLAM.receiver import Receiver
+from multiprocessing import Queue
+from SLAM.globalizer import globalize_points, globalize_landmarks
+from SLAM.sender import send_points, send_position
+
+# creates point objects
+def create_points(point_batch):
+    points = []
+    for p in point_batch:
+        points.append(SLAM.core.Point(p[0], p[1]))
```

Comment:
```suggestion
        points.append(Point(p[0], p[1]))
```
You never imported SLAM itself, therefore this line will fail.